cnxpublishing/views/admin.py

```diff
@@ -310,20 +478,29 @@ def get_baking_statuses_sql(get_request):
         sql_filters = ""
 
     statement = """
-                SELECT m.name, m.authors, m.uuid, m.print_style,
-                       ps.fileid as latest_recipe_id,  m.recipe as recipe_id,
+                SELECT m.name, m.authors, m.uuid,
+                       module_version(m.major_version,m.minor_version)
+                          as current_version,
+                       m.print_style,
+                       CASE WHEN f.sha1 IS NOT NULL
+                       THEN coalesce(ps.print_style,'(custom)')
+                       ELSE ps.print_style
+                       END AS recipe_name,
+                       ps.tag as recipe_tag,
+                       coalesce(dps.fileid, m.recipe) as latest_recipe_id,
+                       m.recipe as recipe_id,
                        f.sha1 as recipe,
-                       module_version(lm.major_version, lm.minor_version)
-                        as latest_version,
-                       module_version(m.major_version, m.minor_version)
-                        as current_version,
                        m.module_ident,
                        ident_hash(m.uuid, m.major_version, m.minor_version),
-                       bpsa.created, bpsa.result_id::text
+                       bpsa.created, ctm.status as state, ctm.traceback
                 FROM document_baking_result_associations AS bpsa
                 INNER JOIN modules AS m USING (module_ident)
+                LEFT JOIN celery_taskmeta AS ctm
```

Comment:
I was tempted to use the celery tables directly before.. but shouldn't we only use the celery APIs?

Reply:
I don't think it is critical to the fix, is it?

Reply:
See the large commit message for my justification: https://github.com/Connexions/cnx-publishing/pull/201/commits/4418b8bba6706704700b290e37ac2e7665a46a27

Reply:
I haven't actually looked at the code on Celery for AsnycResults, but my guess is that it's designed around a memory store (they encourage not bothering with results at all, just discarding them), and that it isn't well optimized for a persistent DB backend. I know I saw a bunch of short connections to the DB while the main query for this template sat in `idle in transaction`.